website/data/docs-nav-data.json

```
@@ -483,6 +483,40 @@
             "title": "Overview",
             "path": "connect/gateways"
           },
+          {
+            "title": "API Gateways",
+            "routes": [
+              {
+                "title": "Overview",
+                "path": "connect/gateways/api-gateways/overview"
```

I would make these singular to be consistent w/ our other gateway types

```suggestion
                "path": "connect/gateways/api-gateway/overview"
```